src/renderers/dom/shared/DOMPropertyOperations.js

```diff
@@ -256,23 +237,19 @@ var DOMPropertyOperations = {
    * @param {string} name
    */
   deleteValueForProperty: function(node, name) {
-    var propertyInfo = DOMProperty.getPropertyInfo(name);
-    if (propertyInfo) {
-      var mutationMethod = propertyInfo.mutationMethod;
-      if (mutationMethod) {
-        mutationMethod(node, undefined);
-      } else if (propertyInfo.mustUseProperty) {
-        var propName = propertyInfo.propertyName;
-        if (propertyInfo.hasBooleanValue) {
-          node[propName] = false;
-        } else {
-          node[propName] = '';
-        }
+    var mutationMethod = DOMProperty.getMutationMethod(name);
+    if (mutationMethod) {
+      mutationMethod(node, undefined);
```

Comment:
Semi-unrelated, but method that this gets called for is `value`, which has this null guard:

```
if (value == null) {
  return node.removeAttribute('value');
}
```

I'm not sure if there is a big distinction between null and undefined in other parts of the code, but passing `null` here, like `mutationMethod(node, null)` feels better to me. What do you think?

Reply:
Really, I think if we stop syncing the value attribute with the value property, we can just drop mutation methods all together.

Reply:
We could just not pass the second argument at all since it will default to `undefined` anyways, but I would prefer `null` here as well. It's also shorter 😄

Reply:
> Really, I think if we stop syncing the value attribute with the value property, we can just drop mutation methods all together.

I wonder if we could just inline the `getMutationMethod` calls as well, since there's only one property that uses mutation methods.

```js
if (name === "value") {
  setDOMValueAttribute(node, null)
}
```